src/facade/facade_types.h

```diff
@@ -34,7 +34,9 @@ struct CmdArgListFormatter {
 
 struct ConnectionStats {
   absl::flat_hash_map<std::string, uint64_t> err_count_map;
-  absl::flat_hash_map<std::string, uint64_t> cmd_count_map;
+  absl::flat_hash_map<std::string, uint64_t> cmd_count_map;  // number of calls per command
```

Comment:
I do not think this map should be here. I think its place is in ServerState data structure because 
we do not need to store a map per connection.

Reply:
Definitely possible. Should the other maps (`err_count`, `cmd_sum`) also move? These maps are only used for metrics so the change won't be problematic.

I guess a reason on why you do want these maps per connection is to avoid "leaking" command history between different users.

Reply:
Dragonfly is a single tenant server and we do not publish per connection stats anywhere. 
 `err_count_map` must be in ConnectionStats because it's updated in dragonfly_connection which belongs to facade lib.
facade lib does not depend on dragonfly_lib. 
On the other hand, cmd_count&sum are updated in DispatchCommand that accesses ServerState hence we can be more efficient there.